scripts/prepare_changelog.sh

```diff
@@ -12,7 +12,8 @@ is_doc_pr(){
         return 1
     fi
     PR_NUM=$1
-    out=$(curl -fsS "https://api.github.com/repos/hashicorp/packer/issues/${PR_NUM}" | jq '[.labels[].name == "docs"] | any')
+    out=$(curl -fsS "https://api.github.com/repos/hashicorp/packer/issues/${PR_NUM}" \
+      | jq '[.labels[].name == "docs" or .labels[].name == "tech-debt"] | any')
```

Comment:
What do you think about changing the name of this method from `is_doc_pr` to `is_doc_or_tech_debt_pr` as well?